pyproject.toml

```diff
@@ -20,8 +20,6 @@ pylint = "^2.14.3"
 pre-commit-hooks = "^4.3.0"
 ruff = "^0.0.269"
 
-[tool.poetry.dev-dependencies]
```

Comment:
how does removing this resolve these security issues?

Reply:
It doesn't, that's just leftovers from some other PR.

What resolves it is the `poetry lock` command which updates the `poetry.lock` file with the latest version of packages that match the version ranges defined above.

Reply:
ahh gotcha